npp.py

```diff
@@ -12,26 +12,50 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-## ------- Set up our list of numbers -------
-S = [25, 7, 13, 31, 42, 17, 21, 10]
-
-## ------- Set up our QUBO dictionary -------
+## ------- import packages -------
+from dwave.system import DWaveSampler, EmbeddingComposite
 
 # TODO:  Add code here to define your QUBO dictionary
+def define_QUBO(S):
```

Comment:
Perhaps just a docstring here? That way a student will know right away what type of data structure `S` is.

Generally, functions are all in snake-case. (i.e. lower case and separated by underscores). https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

```
def define_qubo(S):
    ```Returns a dictionary representing a QUBO.

    Args:
        S(list of integers): represents the numbers being partitioned
    ```
    Q = {}
```